alpha-diversity.rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "alpha-diversity.rmd"
```

Comment:
Let's not use dash in file names, and let us unify file naming, we have used "Rmd" elsewhere.

Change the file names "alpha-diversity.rmd" -> "alpha_diversity.Rmd" and same for other Rmd files.

Reply:
Is this done now?